opencti-platform/opencti-front/src/private/components/settings/users/GroupUsersLines.tsx

```diff
@@ -94,14 +96,15 @@ const GroupUsersLines: FunctionComponent<GroupUsersLinesProps> = ({
     linesFragment: groupUsersLinesFragment,
     queryRef,
   });
+
   const membersData = data.group?.members;
   return (
     <ListLinesContent
       initialLoading={!data}
       isLoading={isLoadingMore}
       loadMore={loadMore}
       hasMore={hasMore}
-      dataList={membersData?.edges ?? []}
+      dataList={groupData.edges ?? []}
```

Comment:
You can't call `groupData.edges` and type `groupData: string[]`, TypeScript won't be happy with that 👿

But more globally, I'm not sure this is the approach we want to fix the issue. By giving members in props you are bypassing Relay refetchable fragment defined in this file. Or if we want to do this that way, you need to remove query that is not used  anymore in this file. @Kedae any sharing on this?

Reply:
Well, it's working but I think we can update the connection from an outside component. I'll check because I think we have done that for tasks. I'll get back to you.

Reply:
As discussed with @CelineSebe > We need to use insertNode and deleteNode but we need to find a way to propagate the paginationOptions